### DISCUSSION TITLE
Breakout room 9

### DISCUSSION THREAD

**Discussion by joemccallum** [IBAHCM/RPiR#208] [Archive]
Joe McCallum - @joemccallum

Kat Hopkinson - @kitkatr

Holly Niven - @hollyisbella

Chuqiao Li - @chuqiaoli

**Comment by joemccallum** [comment]
https://github.com/IBAHCM/joemccallumSeries03.git

**Comment by kitkatr** [comment]
https://github.com/IBAHCM/kathopkinsonSeries3.git

**Comment by chuqiaoli** [comment]
https://github.com/IBAHCM/ChuqiaoSeries03

**Comment by hollyisbella** [comment]
https://github.com/IBAHCM/hollyisbellaSeries3
